Resources/Prototypes/_Moffstation/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/antimateriel.yml

```
@@ -11,3 +11,23 @@
         Holy: 50
   - type: StaminaDamageOnCollide
     damage: 60
+
+- type: entity
+  categories: [ HideSpawnMenu ]
+  parent: BulletAntiMateriel
+  id: BulletAntiMaterielRicochet
+  name: bullet (.60 ricochet)
+  components:
+  - type: Projectile
+    damage:
+      types: # Might change damages
+        Piercing: 75
+        Structural: 200
+    penetrationThreshold: 360
+    penetrationDamageTypeRequirement:
+    - Structural
```

```suggestion
  - type: Projectile
    damage:
      types: # Might change damages
        Piercing: 75
        Structural: 200
```
These'll be inherited.